src/deepmol/splitters/multitask_splitter.py

```diff
@@ -0,0 +1,91 @@
+from typing import List, Tuple
+
+from deepmol.datasets import Dataset
+from deepmol.splitters.splitters import Splitter
+from skmultilearn.model_selection import IterativeStratification
+
+
+class MultiTaskStratifiedSplitter(Splitter):
+
+    def split(self, dataset: Dataset, frac_train: float = 0.8, frac_valid: float = 0.1, frac_test: float = 0.1,
+              seed: int = None, **kwargs) -> Tuple[List[int], List[int], List[int]]:
+        """
+        Splits a dataset into train/validation/test using a stratified split.
+
+        Parameters
+        ----------
+        dataset: Dataset
+            Dataset to split
+        frac_train: float
+            Fraction of dataset to use for training
+        frac_valid: float
+            Fraction of dataset to use for validation
+        frac_test: float
+            Fraction of dataset to use for testing
+        seed: int
+            Seed for the random number generator
+        kwargs
+
+        Returns
+        -------
+        train_indexes: List[int]
+            Indexes of the training set
+        valid_indexes: List[int]
+            Indexes of the validation set
+        test_indexes: List[int]
+            Indexes of the test set
+        """
+
+        if frac_valid == 0:
```

Comment:
what if frac_test = 0? will this work fine?

Reply:
everything works because frac_test will never be 0 because this function is called in the **train_valid_test_split** and **train_test_split** and these methods ensure that **frac_test** will never be 0